telemetry/definitions/commonDefinitions.json

```diff
@@ -878,6 +878,26 @@
             "description": "Open a window to view the Cloudfront Invalidation requests",
             "metadata": [{ "type": "result" }]
         },
+        {
+            "name": "cloudfront_deleteDistribution",
+            "description": "Called when user deletes a CloudFront Distribution",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "cloudfront_deleteStreamingDistribution",
```

Comment:
Is there any merit to collapsing these distro metrics, and adding an isStreamingDistribution flag?

Reply:
Discussed offline. As we know better, we can introduce a specific flag or resourceType